modules/core/src/config.ts

```diff
@@ -22,296 +23,52 @@ export interface Tokens {
   };
 }
 
+// Get the list of ERC-20 tokens from statics and format it properly
+const formattedErc20Tokens = coins.filter((coin: BaseCoin) => {
+  return coin instanceof Erc20Coin;
+}).map((token: Erc20Coin): TokenConfig => {
+  return {
+    type: token.name,
+    coin: token.network.type === NetworkType.MAINNET ? 'eth' : 'teth',
```

Comment:
Can just do `token.family`

Reply:
it doesn't looks like there is a ```token.family``` property on the statics coin objects

Reply:
Ah there is but its always 'eth', my bad

Reply:
i can do this:
```    
coin: `${token.network.type === NetworkType.MAINNET ? '' : 't'}${token.family}`,
```

Reply:
```token.family``` gives the mainnet chain only

Reply:
https://github.com/BitGo/statics/blob/master/src/base.ts#L15